library/LuaApi.cpp

```diff
@@ -1318,6 +1318,8 @@ static bool isSiteLoaded() { return World::IsSiteLoaded(); }
 static std::string df2utf(std::string s) { return DF2UTF(s); }
 static std::string utf2df(std::string s) { return UTF2DF(s); }
 static std::string df2console(color_ostream &out, std::string s) { return DF2CONSOLE(out, s); }
+static std::string cp437Upper(std::string s) { return toUpper_cp437(s); }
+static std::string cp437Lower(std::string s) { return toLower_cp437(s); }
```

Comment:
Could you rename these `upperCp437` (or `toUpperCp437`) and similar for `lower` to match the C++ names and existing Lua functions like `getClipboardTextCp437`?